tests/torch/sparsity/movement/test_scheduler.py

```diff
@@ -80,8 +81,8 @@ def test_error_on_wrong_config(self, desc: dict):
              match='`init_importance_threshold` is equal to or greater'),
     ])
     def test_warn_on_improper_config(self, desc: dict, mocker, caplog):
-        with caplog.at_level(logging.WARNING, logger='nncf'):
-            mocker.patch.object(logging.getLogger('nncf'), 'propagate', True)
+        with caplog.at_level(logging.WARNING, logger=nncf_logger.name):
+            mocker.patch.object(nncf_logger, 'propagate', True)
```

Comment:
Can reuse the `nncf_caplog` fixture now like in `tests/common/test_progress_bar.py`, don't forget to import it either at the `tests/torch/conftest.py` level or in this file.